meta-mender-core/recipes-mender/mender-client/mender-client-cpp.inc

```diff
@@ -15,6 +17,8 @@ RRECOMMENDS:mender-update = "mender-flash"
 # back to line based parsing, which is less resilient.
 RRECOMMENDS:mender-update:append = "${@bb.utils.contains('PACKAGECONFIG', 'modules', ' jq', '', d)}"
 
+RDEPENDS:${PN}-dbg:append = " mender-auth-dbg mender-update-dbg"
```

Comment:
So there *is* a `mender-dbg` package? I'm a bit confused, why does it exist if all the files are in the separate dbg packages?

Reply:
(replying both of your comments here)

The default `PACKAGES` ([documentation here](https://docs.yoctoproject.org/ref-manual/variables.html#term-PACKAGES)) will include `mender-dbg` which before this PR was owning the debug symbols for both binaries.

Related, without setting `NOAUTOPACKAGEDEBUG` all the debug symbols will end up in the _first_ package with `-dbg` suffix (mender-auth in our case) so I needed to set that one to manually package the symbols as intended.

I discussed it f2f with Ole and decided to keep the symbols in the default `mender-dbg`, which makes sense. Think also that the sources for both packages go into `mender-src`, so it is more consistent to keep both debug and source into the same package prefix.